superset-frontend/src/dashboard/stylesheets/components/markdown.less

```diff
@@ -45,6 +45,14 @@
   #brace-editor {
     border: none;
   }
+
+  .resizable-container {
+    span {
+      div {
+        z-index: @z-index-above-dashboard-charts;
```

Comment:
_thank you_ for using the variable :D

Reply:
Uff... it's good that's the correct one :)